spec/ivapi/client/account_spec.rb

```diff
@@ -24,6 +24,18 @@
     end
   end
 
+  describe 'account credits' do
+    before(:each) do
+      stub_command('account_credits', count: 10)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.